File: Question 06/Occurrence.java
class Solution {
    public void answer(Object input) {
      String str = (String) input;
      char target = 'a';
                  int count = 0;
  
                  for (int i = 0; i < str.length(); i++) {
                      if (str.charAt(i) == target) {
                          count++;
                      }
                  }
  
                  System.out.println(count);
    }
  }

File: Question 06/Occurrence.js
// write a program to count the occurrence of a character in a string.
 
function answer(str) {
    let target = 'a';
    let count = 0;
    
    for (let i = 0; i < str.length; i++) {
        if (str.charAt(i) === target) {
            count++;
        }
    }
    
    console.log(count);
    }